src/app/components/tokens/index.html

```diff
@@ -0,0 +1,156 @@
+<sky-demo-page
+  pageTitle="Tokens"
+  skyuxModule="SkyTokensModule"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The tokens component displays a series of specified objects for users to interact with. Users can navigate through the list of tokens and remove tokens from the list. The tokens component's includes a message stream to tokens to the list.
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties sectionHeading="Tokens list properties">
+    <sky-demo-page-property
+      propertyName="disabled"
+      defaultValue="true"
+      isOptional="true"
+    >
+      Indicates whether to disable the tokens list to prevent users from selecting tokens, dismissing tokens, or navigating through the list with the arrow keys. When the tokens list is disabled, users can still place focus on items in the list using the Tab key.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="dismissible"
+      defaultValue="true"
+      isOptional="true"
+    >
+      Indicates whether users can remove tokens from the list by selecting their close buttons or pressing the Backspace key.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="displayWith"
+      defaultValue="name"
+      isOptional="true"
+    >
+      Specifies the token property to display for each item in the tokens list.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="focusable"
+      defaultValue="true"
+      isOptional="true"
+    >
+      Indicates whether users can focus on items in the list using the Tab key. This does not affect the ability of users to select tokens, dismiss tokens, or navigate through the list with the arrow keys.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="messageStream"
+    >
+      Specifies an observable of <stache-code>SkyTokensMessageType</stache-code> that can place focus on a particular token or remove the active token from the list.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="tokens"
+    >
+      Specifies an array of tokens to include in the list. 
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="Tokens list events">
+    <sky-demo-page-property
+      propertyName="focusIndexOverRange"
+    >
+      Fires when users navigate through the token list with the Tab key and attempt to move past the final token in the list.
```

Comment:
Sometimes it reads "tokens list" and sometimes it reads "token list". Should they be consistent?

Reply:
It should be consistent. Thanks for catching that. I'll update all instances of "token list" to be "tokens list."